docs/site/decorators/Decorators_repository.md

```diff
@@ -40,10 +40,10 @@ Syntax: `@model(definition?: ModelDefinitionSyntax)`
 Model decorator is a class decorator. In LoopBack 4, we inherit the model
 definition format from LoopBack 3, which is described in the
 [Model definition JSON file](https://loopback.io/doc/en/lb3/Model-definition-JSON-file).
-For usage examples, see [Define Models](Repositories.md#define-models)
+For usage examples, see [Define Models](Repositories#define-models)
 
 _Please note we will elaborate more about model and model definition in the
-[Model](Model.md) page, and replace the link above with a LoopBack 4 link_
+[Model](Model) page, and replace the link above with a LoopBack 4 link_
```

Comment:
Please revert this. We want to make sure it's rendered correctly on github and loopback.io.

Reply:
@raymondfeng 
Done :white_check_mark:

Reply:
Sorry for the confusion. I think we should change it to `../Model.md`.